src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/covers/ts.ui.CoverSpirit.js

```diff
@@ -11,22 +11,21 @@ ts.ui.CoverSpirit = (function using(chained, Client) {
 	return ts.ui.Spirit.extend(
 		{
 			/**
-			 * Pending a current glitch in Chrome where multiple simultaneous
-			 * transitions freak out, we have disabled cover fades in WebKit.
-			 * UPDATE: We're pushing our luck with WebKit again...
+			 * TODO: remove 'ts-transition' after @dsp has deployed the Tax Selector with new scoped styles!
```

Comment:
WATOOT

Reply:
Did you seriously https://www.urbandictionary.com/define.php?term=WATOOT?

Reply:
🤚